topics/howtosim.rst

```diff
@@ -46,83 +47,79 @@ Parameters for ``gendisk`` are specified in a YAML file with the following field
    DIVERGE_RFAC         : 1                    # Extrapolation exponent for primary mass model
    DIVERGE_RFAC2        : 1                    # Extrapolation exponent for number model
-   DMFAC                : 0.0                  # Disk mass scaling factor for spherical deprojection model
-   DR_DF                : 5                    # Width of change for to Eddington
+   DMFAC                : 1.0                  # Disk mass scaling factor for spherical deprojection model
+   DR_DF                : 1.0                  # Width of change for to Eddington
    DUMPCOEF             : false                # Dump coefficients
+   ASCALE               : XXXXX                # radial scale length for disk basis construction (e.g. 0.01)
    HSCALE               : XXXXX                # Vertical scale length for disk basis construction (e.g. 0.002)
-   LMAX                 : 6                    # Number of harmonics for Spherical SL for halo/spheroid
-   LMAXFID              : 64                   # Number of harmonics for Spherical SL for determining disk basis
+   LMAX                 : 18                   # Number of harmonics for Spherical SL for halo/spheroid
+   LMAXFID              : 128                  # Number of harmonics for Spherical SL for determining disk basis (may want to bump to 256)
    LOGR                 : true                 # Make a logarithmic coordinate mapping
    MMAX                 : 6                    # Number of azimuthal harmonics for disk basis
-   NCHEB                : 16                   # Chebyshev order for smoothing
-   NDP                  : 1
+   CHEBY                : false                # Chebyshev smoothing
+   NCHEB                : 12                   # Chebyshev order for smoothing if CHEBY = true
+   NDP                  : 8                    # Number of angular wedges for disc 
    NDR                  : 2000                 # Number of points in DiskHalo radial table for disk
-   NDZ                  : 200                  # Number of points in DiskHalo vertical table for disk
+   NDZ                  : 800                  # Number of points in DiskHalo vertical table for disk
    NHR                  : 800                  # Number of points in DiskHalo radial table for halo
-   NHT                  : 200                  # Number of points in DiskHalo cos(theta) table for halo
+   NHT                  : 800                  # Number of points in DiskHalo cos(theta) table for halo
    NMAXH                : 18                   # Number of radial basis functions in Spherical SL for halo/spheroid
-   NMAXFID              : 72                   # Number of radial basis functions in Spherical SL for determining disk basis
-   NODD                 : 8                    # Number of vertically antisymmetric disk basis functions per M-order
+   NMAXFID              : 64                   # Number of radial basis functions in Spherical SL for determining disk basis (may want to bump to 128)
```

Comment:
Tests largely again showed that larger than 72 was overkill; but this also isn't as expensive, so it's fine.